release-note-generation/src/main/java/com/google/cloud/ReleaseNoteGeneration.java

```diff
@@ -182,6 +182,9 @@ private void printKeyCoreLibraryDependencies(Bom bom) {
         .append("- Google Cloud Core: ")
         .append(versionlessCoordinatesToVersion.get("com.google.cloud:google-cloud-core"))
         .append("\n");
+    report
```

Comment:
I was considering adding a unit test for it, but looks like we don't have any for this `Core Library Dependencies` section.